### ISSUE TITLE
Is this use case supported?

### ISSUE THREAD

**Issue by s445203** [kfuchs/permission2categories#1]
Hi Kirill,
First off apologies - I have not programmed for a loooong time, and so this is a first time Github post, questions2answer use, permissions2categories use all in one day.  I do apologies if this isn't the best way to ask the question I have - if so please point me in the right direction.

Second off, a huge thanks for taking the effort of making permissions2categories public!

My question is whether what I'm trying to do is actually supported by permissions2categories plugin or not.  

Basically I want some categories to only support creation/editing/deleting/recategorizing actions by higher privileged users.  Is this possible?  Out of the box, permissions2categories just hides entire categories from lower privilege users - whereas I want everyone to be able to read everything.  I have played around but couldn't get it to work - perhaps if I rolled my sleeves up and did something inside qa_categorymetas?  Thanks in advance for any help.

Stephan

**Comment by kfuchs** [comment]
It was absolutely the right way to get in contact with me. But seems like I completely missed the email :( sorry about that. But I guess better late then never...

No you don't have fine grained permissions the way you want. To be honest I have not taken the time to continue support for this project. Hopefully you found some use for it.
